chia/types/blockchain_format/program.py

```diff
@@ -221,8 +221,7 @@ def _run(self, max_cost: int, flags, *args) -> Tuple[int, Program]:
             max_cost,
             flags,
         )
-        # TODO this could be parsed lazily
-        return cost, Program.to(sexp_from_stream(io.BytesIO(ret), SExp.to))
+        return cost, Program.to(ret)
```

Comment:
the main new feature is that we don't need to explicitly deserialize the result here